Packs/Jira/Integrations/JiraV2/JiraV2.yml

```diff
@@ -32,10 +29,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: |-
-    __________________ OAuth 1.0 __________________
-
-    ConsumerKey
+- display: Consumer Key
```

Comment:
maybe add an additionalinfo to explain it is used in oauth1.0?

Reply:
@bakatzir Not supported in 5.0 (maybe also 5.5 don't remember)